etc/DependencyInstaller.sh

```diff
@@ -494,21 +489,17 @@ EOF
         spdlogFolder="/usr/local/lib/cmake/spdlog/spdlogConfigVersion.cmake"
         export spdlogFolder
         _installUbuntuRuntime "${version}"
-        if [[ "${option}" == "dev" ]]; then
-            _installUbuntuDev
-            _installCommonDev
-        fi
+        _installUbuntuDev
+        _installCommonDev
         _installOrTools "ubuntu" "${version}" "amd64"
         _installUbuntuCleanUp
         ;;
     "Red Hat Enterprise Linux")
         spdlogFolder="/usr/local/lib64/cmake/spdlog/spdlogConfigVersion.cmake"
         export spdlogFolder
         _installRHELRuntime
-        if [[ "${option}" == "dev" ]]; then
-            _installRHELDev
-            _installCommonDev
-        fi
+        _installRHELDev
```

Comment:
The goal of this PR should also include replacing both`_install*Runtime` and `_install*Dev` with a single function. Bonus if we can remove any unnecessary packages.